lib/SPIRV/OCLToSPIRV.cpp

```diff
@@ -912,16 +912,16 @@ void OCLToSPIRVBase::transBuiltin(CallInst *CI, OCLBuiltinTransInfo &Info) {
     Info.UniqName = getSPIRVFuncName(BVKind);
   } else
     return;
-  auto Mutator = mutateCallInst(CI, Info.UniqName + Info.Postfix);
+  BuiltinCallMutator Mutator = mutateCallInst(CI, Info.UniqName + Info.Postfix);
   Info.PostProc(Mutator);
+  Type *OldRetTy = CI->getType();
```

Comment:
Do we need to move this definition from if-condition? My guess is that we can leave it as it was

Reply:
True.